recipes/morningstar-data/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "morningstar-data" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: morningstar-data 
+  version: 1.3.0 
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/morningstar_data-{{ version }}.tar.gz
+  sha256: f00509b705558bb452e3f1c154a2b029db9a6c9fa45403f06c6cdba4896fe074
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >=3.8.0,<4.0.0
+    - pandas >=1.5.3,<1.6.0
+    - typeguard >=2.13.3,<3.0.0
+    - pyjwt >=2.4.0,<3.0.0
+    - boto3 >=1.20.1,<2.0.0
+    - cryptography 40.0.2
```

Comment:
This seems overly tight. As you are part of upstream: please loosen that in future.

Reply:
Updated. Thank you!